package.json

```diff
@@ -28,6 +28,10 @@
         "clean": "rm -rf frontend/dist && mkdir frontend/dist",
         "build": "yarn copy-scripts && yarn build:esbuild",
         "build:esbuild": "node frontend/build.mjs",
+        "editor:update-tsd": "node frontend/editor-update-tsd.mjs",
```

Comment:
Shouldn't this be called in the main build process?

Reply:
well, like I wrote [here](https://github.com/PostHog/posthog/pull/10120#issuecomment-1151580629), basically yes, but that would require adding `yarn typegen:write` to the build process, which makes builds a lot slower. Since this is all flagged anyway, I'm hoping to get the current stuff in and build further.